HeritageHub/heritagehub-client
# Setup Project Structure

# Set Up Project Structure in Client (TypeScript) with React Router v6

**Description:**
This pull request sets up the initial project structure for the `heritage-hub-client` using Vite, React, and TypeScript. It includes the following:

1. Creation of directories for assets, components, pages, services, and styles.
2. Addition of placeholder files in each directory.
3. Basic configuration and updates to `App.tsx` and `main.tsx` using React Router v6.

**Tasks Completed:**
- [x] Create directories for project structure.
- [x] Add placeholder files for components, pages, services, and styles.
- [x] Update `App.tsx` and `main.tsx` with React Router v6.

**How to Test:**
1. Clone the repository.
2. Navigate to the `heritage-hub-client` directory.
3. Run `npm install` to install dependencies.
4. Run `npm run dev` to start the development server.
5. Open your browser and navigate to the provided local development URL (usually `http://localhost:5173`) to verify that the project runs without errors.

**Screenshots:**
(Add any relevant screenshots to demonstrate the project structure and running application)

**Linked Issues:**
- This PR closes #2

**Additional Notes:**
- Ensure that you have Node.js and npm installed on your machine before running the commands.
